2024-07-16 17:20:03,409 INFO    MainThread:483208 [wandb_setup.py:_flush():76] Current SDK version is 0.17.4
2024-07-16 17:20:03,409 INFO    MainThread:483208 [wandb_setup.py:_flush():76] Configure stats pid to 483208
2024-07-16 17:20:03,409 INFO    MainThread:483208 [wandb_setup.py:_flush():76] Loading settings from /home/nvan21/.config/wandb/settings
2024-07-16 17:20:03,410 INFO    MainThread:483208 [wandb_setup.py:_flush():76] Loading settings from /work/flemingc/nvan21/projects/iros_simulator/src/python/double_pendulum/controller/SHAC/wandb/settings
2024-07-16 17:20:03,410 INFO    MainThread:483208 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-07-16 17:20:03,410 INFO    MainThread:483208 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-07-16 17:20:03,410 INFO    MainThread:483208 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'src/python/double_pendulum/controller/SHAC/train.py', 'program_abspath': '/work/flemingc/nvan21/projects/iros_simulator/src/python/double_pendulum/controller/SHAC/train.py', 'program': '/work/flemingc/nvan21/projects/iros_simulator/src/python/double_pendulum/controller/SHAC/train.py'}
2024-07-16 17:20:03,410 INFO    MainThread:483208 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-07-16 17:20:03,410 INFO    MainThread:483208 [wandb_init.py:_log_setup():529] Logging user logs to /work/flemingc/nvan21/projects/iros_simulator/src/python/double_pendulum/controller/SHAC/wandb/run-20240716_172003-2d10vgnb/logs/debug.log
2024-07-16 17:20:03,411 INFO    MainThread:483208 [wandb_init.py:_log_setup():530] Logging internal logs to /work/flemingc/nvan21/projects/iros_simulator/src/python/double_pendulum/controller/SHAC/wandb/run-20240716_172003-2d10vgnb/logs/debug-internal.log
2024-07-16 17:20:03,411 INFO    MainThread:483208 [wandb_init.py:init():569] calling init triggers
2024-07-16 17:20:03,411 INFO    MainThread:483208 [wandb_init.py:init():576] wandb.init called with sweep_config: {}
config: {'do_wandb_logging': True, 'actor_units': [256, 256], 'actor_learning_rate': 0.002, 'actor_learning_rate_schedule': 'constant', 'critic_units': [256, 256], 'critic_learning_rate': 0.0005, 'critic_learning_rate_schedule': 'constant', 'critic_minibatches': 4, 'gae_lambda': 0.95, 'gamma': 0.99, 'tau': 0.995, 'num_steps': 32, 'num_envs': 64, 'max_epochs': 100, 'critic_iterations': 16, 'dt': 0.01, 'device': device(type='cuda')}
2024-07-16 17:20:03,411 INFO    MainThread:483208 [wandb_init.py:init():619] starting backend
2024-07-16 17:20:03,411 INFO    MainThread:483208 [wandb_init.py:init():623] setting up manager
2024-07-16 17:20:03,412 INFO    MainThread:483208 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-07-16 17:20:03,416 INFO    MainThread:483208 [wandb_init.py:init():631] backend started and connected
2024-07-16 17:20:03,422 INFO    MainThread:483208 [wandb_init.py:init():720] updated telemetry
2024-07-16 17:20:03,430 INFO    MainThread:483208 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-07-16 17:20:03,661 INFO    MainThread:483208 [wandb_run.py:_on_init():2402] communicating current version
2024-07-16 17:20:03,737 INFO    MainThread:483208 [wandb_run.py:_on_init():2411] got version response 
2024-07-16 17:20:03,737 INFO    MainThread:483208 [wandb_init.py:init():804] starting run threads in backend
2024-07-16 17:20:06,602 INFO    MainThread:483208 [wandb_run.py:_console_start():2380] atexit reg
2024-07-16 17:20:06,602 INFO    MainThread:483208 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2024-07-16 17:20:06,602 INFO    MainThread:483208 [wandb_run.py:_redirect():2300] Wrapping output streams.
2024-07-16 17:20:06,602 INFO    MainThread:483208 [wandb_run.py:_redirect():2325] Redirects installed.
2024-07-16 17:20:06,605 INFO    MainThread:483208 [wandb_init.py:init():847] run started, returning control to user process
2024-07-16 17:20:16,371 WARNING MsgRouterThr:483208 [router.py:message_loop():77] message_loop has been closed
